notebooks/bigQuery.ipynb

```diff
@@ -0,0 +1,214 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc65c551",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import packages\n",
+    "import pandas as pd\n",
+    "import pandas_gbq\n",
+    "\n",
+    "# extract data from BigQuery\n",
+    "project_id = \"govuk-bigquery-analytics\"\n",
+    "\n",
+    "# document types to ignore i.e. pages not relevant to WUJs\n",
+    "ignore = (\n",
+    "    \"authored_article\",\n",
+    "    \"news_article\",\n",
+    "    \"news_story\",\n",
+    "    \"press_release\",\n",
+    "    \"world_news_story\",\n",
+    "    \"utaac_decision\",\n",
+    "    \"speech\",\n",
+    "    \"case_study\",\n",
+    "    \"raib_report\",\n",
+    "    \"asylum_support_decision\",\n",
+    "    \"policy_paper\",\n",
+    "    \"corporate_report\",\n",
+    "    \"written_statement\",\n",
+    "    \"consultation_outcome\",\n",
+    "    \"closed_consultation\",\n",
+    "    \"maib_report\",\n",
+    "    \"person\",\n",
+    "    \"correspondence\",\n",
+    "    \"employment_tribunal_decision\",\n",
+    "    \"employment_appeal_tribunal_decision\",\n",
+    "    \"tax_tribunal_decision\",\n",
+    "    \"ministerial_role\",\n",
+    "    \"residential_property_tribunal_decision\",\n",
+    "    \"cma_case\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9a07dc8e",
+   "metadata": {},
+   "source": [
+    "### By page hits per page path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "187147c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get page paths with more than 5 page hits in the last 2 weeks,\n",
+    "# ignoring irrelevant pages\n",
+    "sql = \"\"\"\n",
+    "DECLARE start_date STRING DEFAULT \"20211018\";\n",
+    "DECLARE end_date STRING DEFAULT \"20211101\";\n",
+    "\n",
+    "WITH sessions AS (\n",
+    "SELECT DISTINCT\n",
+    "    CONCAT(fullVisitorId, \"-\", CAST(visitId AS STRING)) AS sessionId,\n",
+    "    REGEXP_REPLACE(hits.page.pagePath, r'[?#].*', '') AS pagePath\n",
+    "FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`\n",
+    "CROSS JOIN UNNEST(hits) AS hits\n",
+    "WHERE\n",
+    "    _TABLE_SUFFIX BETWEEN start_date AND end_date\n",
+    "    AND hits.page.pagePath NOT LIKE \"/print%\"\n",
+    "    AND hits.type = 'PAGE'\n",
+    "    AND (SELECT value FROM hits.customDimensions WHERE index = 2) NOT IN {} \n",
+    ")\n",
+    "\n",
+    "SELECT \n",
+    "    pagePath,\n",
+    "    COUNT(sessionId) AS noOfSessions \n",
+    "FROM sessions\n",
+    "GROUP BY pagePath\n",
+    "HAVING noOfSessions >= 5\n",
+    "ORDER BY noOfSessions\n",
+    "\"\"\".format(\n",
+    "    ignore\n",
+    ")\n",
+    "\n",
+    "# create Data Frame\n",
+    "page_paths = pandas_gbq.read_gbq(sql, project_id=project_id)\n",
+    "page_paths.to_csv(\"page_paths.csv\", index=False)"
```

Comment:
We will want the user to save it in the data folder, so this should be (as they will be in the notebooks folder): 
`page_paths.to_csv("..data/processed/page_paths.csv", index=False)
`